src/elements/form/Control.ts

```diff
@@ -95,11 +95,15 @@ import { Helpers } from './../../utils/Helpers';
                     <novo-quick-note *ngSwitchCase="'quick-note'" [formControlName]="control.key" [placeholder]="control.placeholder" [config]="control.config" (change)="modelChange($event)"></novo-quick-note>
                 </div>
             </div>
+            <div class="text-field-character-limit" [attr.data-automation-id]="text-area-character-count" *ngIf="_focused && control.maxlength && (control.controlType=='text-area' || control.controlType=='textbox')">
```

Comment:
Styling for this is off, UX wanted it to read `LENGTH/MAX_LENGTH` and be over on the right side, on the same line as the error.

![screenshot 2017-01-19 14 38 38](https://cloud.githubusercontent.com/assets/870799/22124371/38af94d0-de55-11e6-8fca-ecf621e19faa.png)

should resemble

![screenshot 2017-01-19 14 41 16](https://cloud.githubusercontent.com/assets/870799/22124413/60ab73f0-de55-11e6-93ba-e567c0affda3.png)

and turn red when in error too like

![screenshot 2017-01-19 14 41 24](https://cloud.githubusercontent.com/assets/870799/22124424/6924bc6c-de55-11e6-9448-5e9b6f322e22.png)

with the error showing up to the left and the count to the right, all on the same line